style.css

```diff
@@ -1,7 +1,356 @@
 .basis {
+  background-color: #e3eafe;
   font-family: "Roboto", serif;
   font-weight: 900;
-  height: 2766px;
-  width: 1440px;
+  margin: 0;
+  max-width: 1440px;
+  min-width: 300px;
+}
+
+@media screen and (max-width: 320px) {
+  .basis {
+    max-height: 1635px;
+    max-width: 320px;
+  }
+}
+
+.header {
+  background-image: url(images/nav/pic-bgr.jpg);
+  margin-bottom: 22px;
+  min-height: 334px;
+}
+
+@media screen and (max-width: 320px) {
+  .header {
+    background-image: url(images/nav/adapt-bgr.jpg);
+    max-height: 334px;
+    max-width: 320px;
+  }
+}
+
+.nav {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  margin-bottom: 131px;
+}
+
+.nav__link {
+  color: #fff;
+  font-size: 18px;
+  margin-top: 70px;
+  text-decoration: none;
+}
+
+.nav__link:first-child {
+  height: 21px;
+  padding-right: 60px;
+}
+
+.nav__link:nth-child(2) {
+  padding-right: 60px;
+}
+
+.nav__link:nth-child(3) {
+  padding-right: 60px;
+}
+
+.nav__link:nth-child(4) {
+  margin-right: 130px;
+}
+
+.nav__link:hover {
+  color: #fff;
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+.burger {
+  background-color: #fff;
+  /* border: 3px solid #fff; */
+  display: none;
+  height: 3px;
+  position: relative;
+  width: 23px;
+}
+
+.burger::before {
+  background-color: #fff;
+  /* border: 3px solid #fff; */
+  content: " ";
+  height: 3px;
+  position: absolute;
+  top: 5px;
+  width: 23px;
+}
+
+.burger::after {
+  background-color: #fff;
+  /* border: 3px solid #fff; */
+  bottom: 5px;
+  content: " ";
+  height: 3px;
+  position: absolute;
+  width: 23px;
+}
+
+@media screen and (max-width: 320px) {
+  .nav { 
+    display: block;
+    margin-bottom: 15px;
+  }
+
+  .burger {
+    display: flex;
+    float: right;
+    margin-bottom: 15px;
+    margin-right: 29px;
+  }
+}
+
+.introduce {
+  color: #e1e6e9;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  font-weight: 900;
+  justify-content: flex-end;
+  /* margin-bottom: 570px; */
+  margin-right: 130px;
+}
+
+.introduce__title {
+  align-self: flex-end;
+  font-size: 40px;
+  margin-bottom: 11px;
+  margin-right: 0;
+  margin-top: 0;
+}
+
+.introduce__paragraph {
+  align-self: flex-end;
+  font-size: 20px;
+  margin: 0;
+  margin-bottom: 39px;
+  text-align: right;
+  white-space: pre-line;
+}
+
+.button {
+  background-color: #ffda45;
+  border: 0;
+  box-shadow: 5px 5px 20px rgba(0, 0, 0, .25);
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 900;
+  height: 55px;
+  text-align: center;
+  width: 200px;
+}
+
+.button_mediumvioletred {
+  align-self: flex-end;
+  background-color: #e0036f;
+  color: #fff;
+}
+
+@media screen and (max-width: 320px) {
+  .introduce {
+    color: #e1e6e9;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    font-weight: 900;
+    justify-content: center;
+    margin: 0;
+    margin-bottom: 22px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6289680)